godotproject/enemy-Ant.gd

```diff
@@ -0,0 +1,89 @@
+extends KinematicBody2D
+
+signal fireball_start
+
+const GRAVITY = 10
```

Comment:
should each entity have its own gravity or should this be a level wide global?

Reply:
Should be global, I guess the way to do this is to put it in the root node and access root node for gravity. Either in some game config node or level node. I'm going to mark this as resolved for now and create a ticket for this to be added in.

Reply:
Godot seems to have some stuff built in to deal with this: https://docs.godotengine.org/en/stable/classes/class_area2d.html#class-area2d-property-gravity

Not entirely sure how it works or how we use it tho